src/de/grobox/liberario/activities/MainActivity.java

```diff
@@ -91,6 +98,7 @@
 	private AccountHeader accountHeader;
 	private Toolbar toolbar;
 
+	@TargetApi(Build.VERSION_CODES.N_MR1)
```

Comment:
Why did you add this annotation? This method should work on other API levels as well, right?

If you want, you could put the shortcut creation code into its own method and annotate it like this.